moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/trajectory_monitor.h

```diff
@@ -54,9 +54,28 @@ MOVEIT_CLASS_FORWARD(TrajectoryMonitor);  // Defines TrajectoryMonitorPtr, Const
 class TrajectoryMonitor
 {
 public:
+  /**
+   * @brief      This class contains the rcl interfaces for easier testing
+   */
+  class MiddlewareHandle
+  {
+  public:
+    /**
+     * @brief      Destroys the object.
+     */
+    virtual ~MiddlewareHandle() = default;
+
+    /**
+     * @brief      Add sleep using rate
```

Comment:
What is `rate`? I don't see that anywhere else in the public interface or documentation for `MiddlewareHandle`

Reply:
Specify the unit (Hz).

Reply:
No, my point is that in this interface, there is no discussion of rate, sampling frequency, or anything like that. Furthermore, `sleep` could be implemented without respect to a ROS rate, so a more abstract comment like
```
/**
 * @brief      Sleep the handle for some prescribed amount of time.
 */
 virtual void sleep() = 0;
```